content/en/blog/2022/istioctl-proxy/index.md

```diff
@@ -0,0 +1,172 @@
+keywords: [istioctl, cli, external, remote, multicluster]
+---
+
+When using the `istioctl` CLI on a {{< gloss >}}remote cluster{{< /gloss >}} of an
+[external control plane](/docs/setup/install/external-controlplane/) or a [multicluster](/docs/setup/install/multicluster/)
+Istio deployment, some of the commands will not work by default. For example, `istioctl proxy-status` requires access to
+the `istiod` service to retrieve the status and configuration of the proxies it's managing. If you try running it on a
+remote cluster, you'll get an error message like this:
+
+{{< text bash >}}
+$ istioctl proxy-status
+Error: unable to find any Istiod instances
+{{< /text >}}
+
+Notice that the error message doesn't just say that it's unable to access the `istiod` service, it specifically mentions
+its inability to find `istiod` instances. This is because the `istioctl proxy-status` implementation needs to retrieve
+the sync status of not just any single `istiod` instance, but rather all of them. When there is more than one `istiod`
+instance (replica) running, each instance is only connected to a subset of the service proxies running in the mesh.
+The `istioctl` command needs to return the status for the entire mesh, not just the subset managed by one of the instances.
+
+In an ordinary Istio installation where the `istiod` service is running locally on the cluster
+(i.e., a {{< gloss >}}primary cluster{{< /gloss >}}), the command is implemented by simply finding all of the running
+`istiod` pods, calling each one in turn, and then aggregating the result before returning to the user.
+
+{{< image width="75%"
+    link="istioctl-primary-cluster.svg"
+    caption="CLI with local access to istiod pods"
+    >}}
+
+When using a remote cluster, on the other hand, this is not possible since the `istiod` instances are running outside
+of the mesh cluster and not accessible to the mesh user. The instances may not even be deployed using pods on a Kubernetes
+cluster.
+
+Fortunately, `istioctl` provides a configuration option to address this issue.
+You can configure `istioctl` with the address of an external proxy service that will have access to the
+`istiod` instances. Unlike an ordinary load-balancer service, which would delegate incoming requests to one of the
+instances, this proxy service must instead delegate to all of the `istiod` instances, aggregate the responses,
+and then return the combined result.
+
+If the external proxy service is, in fact, running on another Kubernetes cluster, the proxy implementation code
+can be very similar to the implementation code that `istioctl` runs in the primary cluster case, i.e., find all of the
+running `istiod` pods, call each one in turn, and then aggregate the result.
+
+{{< image width="75%"
+    link="istioctl-remote-cluster.svg"
+    caption="CLI without local access to istiod pods"
+    >}}
+
+An Istio Ecosystem project that includes an implementation of such an `istioctl` proxy server can be found
```

Comment:
You use `external proxy service` in the previous paragraphs. Should you have `external` here as well? I also notice you just use proxy service later in the example, so maybe this is OK as it matches them.

Reply:
Yeah, I think it's fine. It's an proxy server for `istioctl` that is deployed on the `external` cluster, so either modifier over `proxy server` should be clear.